data/human/free worlds 0 prologue.txt

```diff
@@ -276,6 +276,321 @@ mission "Pact Questioning"
+	source "Longjump"
+	destination "New Portland"
+	"apparent payment" 55000
+	cargo "farming equipment" 20
+	to offer
+		not "event: chosen sides"
+		has "Pact Supplies 0: done"
+	blocked `You need <capacity> before you can partake in the next stage of the convoy for the <Pact/Free Worlds>.`
+	on offer
+		conversation
+			branch bombings
+				has "event: war begins"
+			action
+				set "pact supplies heard argument"
+			`You return to the spaceport to find the militia officer ordering forklifts laden with crates into ships while speaking with a few of the other captains you recognize from the convoy. He turns and notices you as you approach.`
+			`	"Alright, for this stage of the convoy we will be heading to <planet>. You will be transporting farming equipment, which we will trade with the global farming cooperatives there for food supplies that we will bring back to <origin>. The militias on <planet> have decided to send excess food in lieu of weaponry and ship equipment as they run a smaller planetary militia than most, but we have a good relationship with the farmers and their militia would like to support us anyway.`
+			`	"You'll be paid <payment> for your work, of which half will be deposited when you reach <planet> and the other half when you return here to <origin>. Is that clear?"`
+			choice
+				`	"Yes."`
+				`	"I've changed my mind. I don't want to do participate in this convoy any more."`
+					decline
+
+			`	"Brilliant. I pray that you have a safe journey."`
+				accept
+
+			label bombings
+			`You return to the spaceport to find the militia officer in what appears to be a rather loud discussion with a couple of the other captains from the convoy. He doesn't seem to notice you as you approach.`
+			`	"I assure you, the Pact - and what is now the Free Worlds - has absolutely nothing to do with the bombings of Martini and Geminus. This is as much of a shock and sense of despair to us down in the Southern Rim as it is must be for those in the Paradise Worlds. But it would be unconscionable for the leaders of the Pact to do such a thing. I can't imagine the Pact Council authorizing such a thing - we'd reduce ourselves to the level of the pirates we seek to eliminate from the galaxy."`
+			`	The two captains frown. "Like it or not, this is all very good timing for your separarist leaders," the shuttle captain says. "And I don't plan to be barred from returning to my home planet in Cor Caroli when the Navy inevitably comes down to reclaim your systems. I have family there to visit, to send money back to, you know, and I don't plan to be labeled a separatist and shot out of the sky by the Reps because I took on a little convoy mission to what I thought was a law-abiding organization. Goodbye." The captain of one of the Tarazed Sparrows follows her out.`
```

Comment:
```suggestion
			`	The two captains frown. "Like it or not, this is all very good timing for your separatist leaders," the shuttle captain says. "And I don't plan to be barred from returning to my home planet in Cor Caroli when the Navy inevitably comes down to reclaim your systems. I have family there to visit, to send money back to, you know, and I don't plan to be labeled a separatist and shot out of the sky by the Reps because I took on a little convoy mission to what I thought was a law-abiding organization. Goodbye." The captain of one of the Tarazed Sparrows follows her out.`
```
seperatist*